src/libumf_windows.c

```diff
@@ -33,14 +33,16 @@ void libumfInit(void) {
 
 INIT_ONCE init_once_flag = INIT_ONCE_STATIC_INIT;
 
+static void umfTearDownWrapper(void) { (void)umfTearDown(); }
```

Comment:
Pls add info somewhere, perhaps, that it is related to ICX on Win...?

Reply:
It's not necessary, because it's not related strictly with the icx, the icx compiler just trigger warning.

The rules regarding how to determine if two function types are compatible are spelled out in section 6.7.6.3p15 of the [C standard](https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf):

```
For two function types to be compatible, both shall specify compatible return types.
Moreover, the parameter type lists, if both are present, shall agree in the number of
parameters and in use of the ellipsis terminator; corresponding parameters shall have 
compatible types.
```